app/js/lib/server/reminders.js

```diff
@@ -41,6 +41,17 @@ export default class Reminders {
   }
 
   /**
+   * Create a new reminder.
+   *
+   * @param {Object} body
+   * @return {Promise}
+   */
+  update(body) {
+    const id = body.id;
```

Comment:
I'll check for the existence of id.

The other difference with `set` is the HTTP verb (POST vs. PUT).